grammar.md

```diff
@@ -1,7 +1,12 @@
 # Grammar
 
 ### Pre-Notes
-"?" is the nil-check operator. If the expression gets nil, an error is thrown.
+
+ * "?" is the nil-check operator. If the expression gets nil, an error is thrown.
+
+ * All functions return nil by default.
```

Comment:
To be pedantic, a function _without explicit return values_ will return `nil` be default.

Reply:
I'll fix the typo later!